recipes/apptainer/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: apptainer
+  version: {{ version }}
+
+source:
+  - folder: src/github.com/apptainer/apptainer
+    url: https://github.com/apptainer/apptainer/releases/download/v{{ version }}/apptainer-{{ version }}.tar.gz
+    sha256: 1772f25c8770f976abb82c29053b77183930426a83eb3e43b9f8068c3e88ac30
+    patches:
+      #- patches/0003-Use-external-CNI.patch
+      #- patches/0004-Find-unsquashfs-from-PATH.diff
+      #- patches/set-version-from-env.patch
```

Comment:
This patch will probably need to be rebased, the others should be checked for if they're still needed.